src/plugins/kibana_overview/public/components/overview/overview.tsx

```diff
@@ -67,14 +68,14 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
     docLinks,
     dataViews,
     share,
-    uiSettings,
     application,
     chrome,
     dataViewEditor,
     customBranding,
+    theme,
   } = services;
   const addBasePath = http.basePath.prepend;
-  const IS_DARK_THEME = uiSettings.get('theme:darkMode');
+  const currentTheme = useObservable(theme.theme$, { darkMode: false });
```

Comment:
The logic to retrieve the information of darkMode was already wrong before this PR here (given the per-user-profile darkMode override). Fixed.